android/local.properties

```diff
@@ -0,0 +1,9 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Oh wow I didn't see that :man_facepalming:

I removed the files that weren't checked in in the main repo and added some lines to the .gitignore.

I removed the gradle wrapper files since they also weren't added to the main repo. However, I didn't add those files to the .gitignore since the [docs](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:wrapper_generation) advise against it.